polaris-service/src/test/java/org/apache/polaris/service/catalog/PolarisSparkIntegrationTest.java

```diff
@@ -264,6 +268,20 @@ public void testCreateTable() {
     assertThat(recordCount).isEqualTo(3);
   }
 
+  @Test
```

Comment:
nit: new tests should probably go at the end of the file

Reply:
IMO, we should group tests for the same entities together. This will improve readability.
WDYT?

Reply:
Totally agreed -- but this test to create a view is between a test for creating a table and one for creating an external table. 

It's not obvious to me that this keeps tests for the same entities together any better than if the test for the view was moved to the end

Reply:
makes sense. Thanks.